scripts/partition_manager_report.py

```diff
@@ -48,6 +48,16 @@ def print_region(region, size, pm_config):
                 hex(pm_config[name]['size']),
                 bcolors['ENDC'])
             for name in sorted_pm_config]
+    endspan = 0
+    i = 0
+    for name in sorted_pm_config[:-1]:
+        i += 1
+        if 'span' in pm_config[name]:
+            endspan = pm_config[name]['end_address']
+        if pm_config[name]['end_address'] == endspan and not ('span' in pm_config[name]) \
+                and not 'span' in pm_config[sorted_pm_config[sorted_pm_config.index(name)+1]]:
```

Comment:
I get the following flake8 warnings:
```
../scripts/partition_manager_report.py:57:80: E501 line too long (90 > 79 characters)
../scripts/partition_manager_report.py:58:21: E713 test for membership should be 'not in'
../scripts/partition_manager_report.py:58:80: E501 line too long (94 > 79 characters)
```

(The line length may be ignored as we have no official rule for python code, but I still find it nice to validate code with flake8)